api/src/main/java/uk/gov/dhsc/htbhf/dwp/controller/v2/DWPEligibilityControllerV2.java

```diff
@@ -36,6 +39,9 @@ public IdentityAndEligibilityResponse getBenefits(@ApiParam("The identity and el
                                                               DWPEligibilityRequestV2 request) {
         log.debug("Received eligibility request");
 
-        return IdentityAndEligibilityResponse.builder().build();
+        IdentityAndEligibilityResponse response = service.checkIdentityAndEligibility(request);
+        log.debug("DWP identity status: {}, eligibility status: {}, qualifying benefits: {} ",
```

Comment:
I was unsure what to log here (if anything). The v1 controller simply outputs the eligibility response, but we have a lot more statuses on the response now so was unsure which to log. Don't want to log the whole object as it has children dobs on it. Thoughts?

Reply:
I think postcode & address matches would be useful as we then have all the statuses that determine whether someone has successfully identified themselves. Qualifying benefits may come in handy in the future, but right now they'll always be UC.